website/docs/tutorials/03-project-initialization/02-protostar-toml-v2.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_label: protostar.toml V2
+---
+# `protostar.toml` V2
+
+```toml title="protostar.toml example"
```

Comment:
This config file example is not really realistic, we should show a real example, which would be useful when creating your own config file.

Reply:
[make the example more real](https://github.com/software-mansion/protostar/pull/1078/commits/432954984a8060190d929f5abeb13ade7b3c0a00)